components/feature/webnotifications/src/main/java/mozilla/components/feature/webnotifications/WebNotificationFeature.kt

```diff
@@ -70,13 +70,13 @@ class WebNotificationFeature(
             notificationManager?.cancel(NOTIFICATION_TAG_ID, it)
         }
 
-        pendingRequestId++
-        notificationId++
+        val notificationId = NotificationIds.getNextIdForTag(context, NOTIFICATION_TAG_ID)
+        val pendingIntentId = NotificationIds.getNextIdForTag(context, PENDING_INTENT_TAG_ID)
         notificationIdMap[webNotification.tag] = notificationId
 
         GlobalScope.launch(Dispatchers.IO) {
             val notification = nativeNotificationBridge.convertToAndroidNotification(
-                webNotification, context, NOTIFICATION_CHANNEL_ID, activityClass, pendingRequestId)
+                webNotification, context, NOTIFICATION_CHANNEL_ID, activityClass, pendingIntentId)
             notificationManager?.notify(NOTIFICATION_TAG_ID, notificationId, notification)
```

Comment:
Actually, can you help me understand this. Since the notification already comes with a unique tag - do we even need to provide an incrementing id? Isnt (tag, 1) already unique?

Reply:
You're correct.  By using the tag given to us through GeckoView, we can use the same notification ID (ex: 1) and everything still works as expected.  I will cancel this review and update the implementation to use the tag from GeckoView.